docs/api/index.md

```diff
@@ -40,11 +40,13 @@ To protect our infrastructure, we enforce rate-limits on the API and the website
 - 100 req/min for all read product queries (`GET /api/v*/product` requests or product page). There is no limit on product write queries.
 - 10 req/min for all search queries (`GET /api/v*/search` or `GET /cgi/search.pl` requests)
 
-If these limits are reached, we reserve the right to deny you the access to the website and the API through IP address ban. If your IP has been banned, feel free to send us an email to explain us the reason why you reached the limits: reverting the ban is possible.
+If these limits are reached, we reserve the right to deny you access to the website and the API through IP address ban. If your IP has been banned, feel free to [email us to explain why you reached the limits][why_reached_limits]: reverting the ban is possible.
 
-If your requests come from your users directly (ex: mobile app), the rate limits will apply per user.
+[why_reached_limits]: mailto:contact@openfoodfacts.org
```

Comment:
```suggestion
[why_reached_limits]: mailto:reuse@openfoodfacts.org
```

Reply:
I can do that ("contact@" was OK'd by @alexgarel). Anything else so I can batch changes in one commit?

Reply:
Done. Let me know if you need me to rebase and or squash.